stores/useVotePluginsClientStore.tsx

```diff
@@ -74,11 +92,13 @@ interface UseVotePluginsClientStore extends State {
 
 const defaultState = {
   vsrClient: undefined,
+  heliumVsrClient: undefined,
   nftClient: undefined,
   gatewayClient: undefined,
   switchboardClient: undefined,
   pythClient: undefined,
   voteStakeRegistryRegistrar: null,
+  heliumVsrRegistrar: null,
```

Comment:
(dont change) I'm amused by the mixed usage of undefined and null in the existing state, ha